bahmnicore-omod/src/main/java/org/bahmni/module/bahmnicore/web/v1_0/search/BahmniConceptSearchHandler.java

```diff
@@ -59,4 +68,20 @@ public PageableResult search(RequestContext context) throws ResponseException {
         }
     }
 
+    private List<Locale> getLocales(RequestContext context) {
```

Comment:
Can you add a comment on this method to explain why this code is written. I guess, to explain that if a concept is not found in user's locale, we search in default locale too -- and hence adding both locales into the locale list.